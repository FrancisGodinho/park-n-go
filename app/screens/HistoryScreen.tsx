import { SafeAreaView, StyleSheet, Text, View } from "react-native";
import React from "react";

type Props = {};

const HistoryScreen = (props: Props) => {
  return (
    <SafeAreaView>
      <Text>HistoryScreen</Text>
    </SafeAreaView>
  );
};

export default HistoryScreen;

const styles = StyleSheet.create({});
