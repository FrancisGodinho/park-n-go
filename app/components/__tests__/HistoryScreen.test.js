import React from "react";
import renderer from "react-test-renderer";
import { cleanup } from "@testing-library/react-native";
import { fireEvent, render } from "@testing-library/react";
import HistoryScreen from "../../screens/HistoryScreen";

let findTextElement = function (tree, element) {
  console.warn(tree);
  return true;
};

describe("<HistoryScreen />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<HistoryScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
