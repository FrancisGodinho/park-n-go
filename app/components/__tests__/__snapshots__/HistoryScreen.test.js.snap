// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HistoryScreen /> renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <Text
    style={
      Object {
        "color": "#C0C0C0",
        "fontFamily": "Avenir-Black",
        "fontSize": 25,
        "fontWeight": "900",
        "marginBottom": 20,
        "textAlign": "center",
      }
    }
  >
    Parking History
  </Text>
  <RCTScrollView>
    <View>
      <Text
        style={
          Object {
            "color": "#C0C0C0",
            "fontFamily": "Avenir-Black",
            "fontSize": 20,
            "fontWeight": "700",
          }
        }
      >
        No Parking History Yet...
      </Text>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
